tsconfig.json

```diff
@@ -18,7 +18,12 @@
     "noFallthroughCasesInSwitch": true,
     "allowSyntheticDefaultImports": true,
     "allowJs": true,
-    "alwaysStrict": true
+    "alwaysStrict": true,
+    "paths": {
+      "gatsby-theme-iterative-docs/*": [
+        "./plugins/gatsby-theme-iterative-docs/*"
+      ]
+    }
```

Comment:
This enables `tsc` linting to work with the faked package imports, and will have to be removed once the theme becomes a real package.